tests/integration/pilot/analysis/main_test.go

```diff
@@ -30,16 +29,6 @@ func TestMain(m *testing.M) {
 	framework.
 		NewSuite(m).
 		RequireSingleCluster().
-		Setup(istio.Setup(nil, func(_ resource.Context, cfg *istio.Config) {
```

Comment:
can be followup - if we do this we should merge into the mainline pilot/ package